XCUITests/FxScreenGraph.swift

```diff
@@ -191,11 +191,15 @@ func createScreenGraph(_ app: XCUIApplication, url: String = "https://www.mozill
 // Invoking this method in BrowserTab will create another tab,
 // as that is the shortest path to itself
 extension Navigator {
-    func openNewURL(urlString: String) {
+    func openNewURL(urlString: String, gotoURL: Bool = true) {
         self.goto(NewTabScreen)
         let app = XCUIApplication()
         app.textFields["url"].tap()
-        app.textFields["address"].typeText(urlString + "\r")
-        self.nowAt(BrowserTab)
+        if gotoURL {
+            app.textFields["address"].typeText(urlString + "\r")
+            self.nowAt(BrowserTab)
+        } else {
+            app.textFields["address"].typeText(urlString)
+        }
     }
```

Comment:
My suggestion was to add an extra scene in the graph, so you could re-use the same tab – so the tests would run faster. I've put those in #2466.

This works, but the function is no longer named accurately: `openNewURL` doesn't actually open a new URL.

Given an `OpenURLBar` state, I'd add `navigator.goto(OpenURLBar)` to your deleted `typeOnSearchBar(text: String)` method and use that.